src/pages/components/CertificateInfo.qml

```diff
@@ -103,146 +105,69 @@ SilicaFlickable {
             }
         }
 
+        DetailItem {
+            //: Label for the issuer field of a TLS certificate
+            //% "Verified by"
+            label: qsTrId("sailfish_browser-la-cert_issuer")
+            value: security ? security.issuerDisplayName : ""
+            visible: value && value.length > 0
+        }
+
```

Comment:
In the figma designs I also see "Connection details" info shown. This is removed later in some of the screenshots, but I don't see any discussion about whether it should be kept/removed on the bug.

Should a "Connection details" line be included here?

Reply:
Yes, they want to include it. But it appeared after I started this work, and still has no agreement

Reply:
Any update on this?

Reply:
https://github.com/sailfishos/sailfish-browser/pull/725
still watinig...